Add hint on removing SUSE products (#2297)
theforeman/foreman-documentation: guides/common/modules/proc_adding-an-scc-account.adoc

= Adding an SCC Account to {Project}

.Prerequisite
* You have installed the SCC Manager plug-in on your {Project}.
For more information, see xref:Installing_the_SCC_Manager_Plugin_{context}[].

.Procedure

> . In the {ProjectWebUI}, navigate to *Content > Content Credentials* and click *Create Content Credential*.

Reviewer: This step should be marked as optional in my opinion.

Author: Applied.

> . Optional: In the {ProjectWebUI}, navigate to *Content > Content Credentials* and click *Create Content Credential*.

Reviewer 2: ```suggestion
. Optional: In the {ProjectWebUI}, navigate to *Content* > *Content Credentials* and click *Create Content Credential*.
```

Author: Applied.

+
Add the GPG public key for SLES 15 SP3 from https://www.suse.com/support/security/keys/[suse.com].

> . In the {ProjectWebUI}, navigate to *Content > SUSE Subscriptions*.

Reviewer 2: ```suggestion
. In the {ProjectWebUI}, navigate to *Content* > *SUSE Subscriptions*.
```

Author: Applied.

. Click *Add SCC Account*.
. In the *Name* field, enter a name for your SCC account in {Project}.
. In the *Login* and *Password* fields, enter your SUSE credentials.
. In the *Base URL* field, you can enter the base URL for the Suse Customer Center.
By default, it is set to `\https://scc.suse.com`.
. Optional: Set a *Sync interval* to periodically update the SCC authentication tokens.
Note that this does not refer to synchronizing content to {Project}.

> . In the *Use GPG key for SUSE products* field, you can select the previously created content credential to automatically add a GPG public key to your SUSE products.

Reviewer: Should this also be marked as optional to match line 9? The default GPG Key selection is `None`

Author: Yes, applied.

`zypper` automatically verifies the signatures of each software package to ensure their authenticity.
+
[NOTE]
====
You can also set the GPG public key for repositories from SUSE at a later stage.
However, changing it does not affect already synchronized products.
If you already have synchronized products in {Project}, navigate to *Content > Products* and replace the GPG key in each respective product.
====

> . In the *Download Policy* drop down menu, you can select a download policy for your SUSE products.

Reviewer 2: ```suggestion
. Optional: from the *Download Policy* list, select a download policy for your SUSE products.
```
The word "can" implies that this step is optional, am I right about this?

Author: Yes, reworded.

> . In the *Download Policy* drop down menu, you can select a download policy for your SUSE products.

Reviewer 2: ```suggestion
. Optional: From the *Download Policy* list, select a download policy for your SUSE products.
```
The word "can" implies that this step is optional, am I right about this?

For more information, see xref:Download_Policies_Overview_{context}[].

> . In the *Mirroring Policy* drop down menu, you can select a mirroring policy for your SUSE products.

Reviewer 2: ```suggestion
. Optional: From the *Mirroring Policy* list, select a mirroring policy for your SUSE products.
```

Author: Reworded.
